contracts/0.4.24/Lido.sol

```diff
@@ -78,6 +81,8 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
     bytes32 internal constant BEACON_BALANCE_POSITION = keccak256("lido.Lido.beaconBalance");
     /// @dev number of Lido's validators available in the Beacon state
     bytes32 internal constant BEACON_VALIDATORS_POSITION = keccak256("lido.Lido.beaconValidators");
+    /// @dev total amount of Ether MEV and transaction rewards received by Lido contract
+    bytes32 internal constant MEV_TX_FEE_ETHER_POSITION = keccak256("lido.Lido.mevTxFeeEther");
```

Comment:
add total 
write that these rewards are not accumulated in contract and not used for logic, it's just a counter